pkg/controller/azdrivernode.go

```diff
@@ -55,8 +55,12 @@ func (r *ReconcileAzDriverNode) Reconcile(ctx context.Context, request reconcile
 	n := &corev1.Node{}
 	err := r.cachedClient.Get(ctx, request.NamespacedName, n)
 
-	// If the node still exists don't delete the AzDriverNode
 	if err == nil {
```

Comment:
If `error != nill` and is not `errors.IsNotFound(err)`, we should probably reconcile because we failed to read from the cacheClient.

Reply:
If `error != nil`l and is not `errors.IsNotFound(err)`, the code will reach the last line of current function:
`return reconcile.Result{Requeue: true}, err`
And `Requeue: true` means it will reconcile which is what we want. Right? Correct me if I misunderstand.

Reply:
Oh, you are right! I was reviewing on github and missed the end of the method.